### ISSUE TITLE
tar to stdout prints "removing leading / from member names"

### ISSUE THREAD

**Issue by justincormack** [landley/toybox#55]
Using `tar` as `tar cf - /bin/sh` or similar, I could not get the archive recognised, doing a hexdump I realised it was outputing the warning to stdout before the archive, which is not very helpful:

```
00000000  72 65 6d 6f 76 69 6e 67  20 6c 65 61 64 69 6e 67  |removing leading|
00000010  20 27 2f 27 20 66 72 6f  6d 20 6d 65 6d 62 65 72  | '/' from member|
00000020  20 6e 61 6d 65 73 0a 62  69 6e 2f 73 68 00 00 00  | names.bin/sh...|
00000030  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
```

**Comment by landley** [comment]
On 11/19/2016 10:06 AM, Justin Cormack wrote:
> Using |tar| as |tar cf - /bin/sh| or similar, I could not get the
> archive recognised, doing a hexdump I realised it was outputing the
> warning to stdout before the archive, which is not very helpful:

I note that "toys/pending" is toybox what "drivers/staging" is to the
linux kernel. It has a README to that effect, and current releases
produce a red warning when you build code out of it.

I would love to get tar out of pending (see
http://landley.net/toybox/cleanup.html for the kind of work that
generally involves), but toybox has never been my day job. I've gotten
to work on it half-time in work contexts a couple times, but the past
few months have been insane with non-toybox work commitments. (I was in
Japan for 2 weeks, flew back in time to vote, got on a plane to
minneapolis 2 days later, got back from that monday, and am currently
scheduled to fly to california on tuesday, and from there probably to
san diego. I might get back to Austin around December 10th, just in time
for a road trip.)

tl;dr I can fix this specific bug (done), but until I find time to clean
up tar and move it out of pending I wouldn't rely on it too heavily.

Rob

(The annoying part is if I do an hour's worth of work without getting to
a good checkin point, and I don't get back to it for a couple days, it
can take an hour of reading and fiddling with the code to refamiliarize
myself with enough context/details to figure out where I left off and
why. I need large contiguous chunks of time/energy to do some kinds of
work, those have all been spoken for recently.)

**Comment by justincormack** [comment]
Sorry, I didn't notice it was in pending, I built `tar` using the nice standalone build mode rather than through the config. Thanks for fixing it, appreciate your time.
